azext_iot/central/_help.py

```diff
@@ -1752,3 +1754,282 @@ def _load_central_edge_help():
+      """
+
+    helps[
+        "iot central scheduled-job list"
+    ] = """
+    type: command
+    short-summary: Get the list of scheduled job definitions in an application
+    examples:
+      - name: Get the list of scheduled job definitions in an application
+        text: >
+          az iot central scheduled-job list
+          --app-id {appid}
+    """
+
+    helps[
+        "iot central scheduled-job show"
+    ] = """
+    type: command
+    short-summary: Get details about a scheduled job by ID
+    examples:
+      - name: Get details about a scheduled job by ID
+        text: >
+          az iot central scheduled-job show
+          --app-id {appid}
+          --id {scheduledJobId}
+    """
+
+    helps[
+        "iot central scheduled-job delete"
+    ] = """
+    type: command
+    short-summary: Delete an existing scheduled job by ID
+    examples:
+      - name: Delete an existing scheduled job by ID
+        text: >
+          az iot central scheduled-job delete
+          --app-id {appid}
+          --id {scheduledJobId}
+    """
+
+    helps[
+        "iot central scheduled-job create"
+    ] = """
+    type: command
+    short-summary: Create a scheduled job by ID
+    examples:
+      - name: Create a schedule job with name
+        text: >
+          az iot central scheduled-job create
+          --app-id {appid}
+          --job-id {jobId}
+          --group-id {groupId}
+          --job-name {displayName}
+          --schedule	{jobScheduleJSONPatch}
+          --content {jobDataJSONPath}
+
+      - name: Create a scheduled job with name and batch configuration.
+        text: >
+          az iot central scheduled-job create
+          --app-id {appid}
+          --job-id {jobId}
+          --group-id {groupId}
+          --job-name {displayName}
+          --schedule	{jobScheduleJSONPatch}
+          --content {jobDataJSONPath}
+          --batch {jobBatchValue}
+          --batch-type {jobBatchType}
+
+      - name: Create a scheduled job with name and cancellation threshold configuration with no batch.
+        text: >
+          az iot central scheduled-job create
+          --app-id {appid}
+          --job-id {jobId}
+          --group-id {groupId}
+          --job-name {displayName}
+          --schedule	{jobScheduleJSONPatch}
+          --content {jobDataJSONPath}
+          --cancellation-threshold {jobCancellationThresholdValue}
+          --cancellation-threshold-type {jobCancellationThresholdType}
+          --description {jobDesc}
+    """
+
+    helps[
+        "iot central scheduled-job update"
+    ] = """
+    type: command
+    short-summary: Update a scheduled job by ID
+    examples:
+      - name: Update a schedule job with name
+        text: >
+          az iot central scheduled-job update
+          --app-id {appid}
+          --job-id {jobId}
+          --job-name {displayName}
+    """
+
+    helps[
+        "iot central scheduled-job list-jobs"
```

Comment:
what is the difference between this list-jobs and list?

Reply:
List is listing scheduled job definition itself and list-jobs is listing scheduled job instances (jobs that already ran, failed, completed). A scheduled job can be triggered multiple times, each time will generate a new job. 👀

https://docs.microsoft.com/en-us/rest/api/iotcentral/2022-07-31dataplane/scheduled-jobs/list-jobs

Reply:
Can we use better wording for command - like list child jobs, job runs, or something else? Is there official central documentation (as in not rest api but something like portal how to) that describes this/how to deal with jobs better (then we can rip the wording from there)

Reply:
Will do, list-runs